migrations/14.24.2-add-api-key-expiry-date.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "api key-has-role" ADD COLUMN "expiry date" TIMESTAMP NULL;
```

Comment:
This should be a migration attached to the user model, ie in https://github.com/balena-io/pinejs/blob/v14.35.1/src/sbvr-api/permissions.ts#L1624-L1653

Reply:
So this would be some like below, after [L1653](https://github.com/balena-io/pinejs/blob/v14.35.1/src/sbvr-api/permissions.ts#L1624-L1653):
```
'14.24.2-modified-at': `
     ALTER TABLE "api key-has-role" ADD COLUMN "expiry date" TIMESTAMP NULL;
```

Reply:
Yes